System.IO.Abstractions.TestingHelpers.Tests/MockFileSystemTests.cs

```diff
@@ -277,5 +277,20 @@ public void MockFileSystem_AddDirectory_TrailingSlashAllowedButNotRequired(strin
 
             Assert.IsTrue(fileSystem.FileExists(path2));
         }
+
+        [Test]
+        public void MockFileSystem_GetFiles_ThrowsCorrectExceptionForInvalidCharacters()
```

Comment:
I prefer to be specific in the naming, so what about that:

```suggestion
        public void MockFileSystem_GetFiles_ThrowsArgumentExceptionForInvalidCharacters()
```

Reply:
Makes sense